tests/performance/config/perf-driver/measure/measure-deploymentTime.yml

```diff
@@ -15,7 +15,24 @@ config:
     - name: deploymentTime
       uuid: cfac77fceb244862aedd89066441c416
       desc: The time from the HTTP request completion till the deployment success
-      summarize: [mean_err]
+      summarize:
```

Comment:
As @meln1k pointed out it probably helps to have min and max as well.

Reply:
Good point 👍

Reply:
Yup, good idea 👍